libretroshare/src/pqi/authssl.cc

```diff
@@ -1127,11 +1127,12 @@ bool AuthSSLimpl::AuthX509WithGPG(X509 *x509, uint32_t& diagnostic)
 			goto err;
 		}
 
-		RsInfo() << __PRETTY_FUNCTION__ << " Verified: " << sigtypestring
-		         << " signature of certificate sslId: "
-		         << RsX509Cert::getCertSslId(*x509)
-		         << ", Version " << std::hex << certificate_version << std::dec
-		         << " using PGP key " << pd.fpr << " " << pd.name << std::endl;
+        if(verbose)
+			std::cerr<< " Verified: " << sigtypestring
```

Comment:
Please keep `RsInfo() << __PRETTY_FUNCTION__ ` instead of `std::cerr`

Reply:
Well, here I removed it because it prints too much. We don't need the time nor the function name.

Reply:
If you don'T want the function name just keep `RsInfo()` at least it will print well on all platforms, plain `std::cXXX` doesn't work on Android so I had to do a dirty trick to be able to get those messages too, but it doesn'T wok always, while if you use the thins defined in `util/rsdebug.h` it works well on all platforms